### DISCUSSION TITLE
Make more space for absolute scroll song selection

### DISCUSSION THREAD

**Discussion by ziomek64** [ppy/osu#17543] [Ideas] [answered]
At first I was like that absolute scroll doesn't work. I was so used to doing that on the middle of screen. Now you have to do it really close to the song list
![osu_2022-03-29_23-27-18](https://user-images.githubusercontent.com/84880432/160710546-5c894316-482e-49ab-9c07-f27a0843e591.jpg)
.
The screenshot shows where you can't scroll now already and when it would be preferable as before. Red line and green line

**Comment by respektive** [comment, marked as answer]
Most likely going to be addressed in an upcoming song select redesign.
Also https://github.com/ppy/osu/issues/2238

**Reply by ziomek64** [reply]
That issue you mentioned should be it. Very specific title, couldn't find it

**Reply by integerrr** [reply]
you couldnt find #17500?
